contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/ArbitrumInboxInterface.sol";
+import "../interfaces/AggregatorValidatorInterface.sol";
+import "../interfaces/FlagsDevInterface.sol";
+import "../SimpleWriteAccessController.sol";
+
+/**
+ * @title ArbitrumValidator Contract
+ * @notice Allows to raise and lower Flags on the Arbitrum network through its Layer 1 contracts
+ * The internal AccessController controls the access of the validate method
+ * Gas configuration is controlled by a configurable external SimpleWriteAccessController
+ * Funds on the contract are managed by the owner
+ */
+contract ArbitrumValidator is SimpleWriteAccessController, AggregatorValidatorInterface {
+  // Follows: https://eips.ethereum.org/EIPS/eip-1967
+  address constant private s_arbitrumFlag = address(bytes20(bytes32(uint256(keccak256("chainlink.flags.arbitrum-offline")) - 1)));
+  bytes constant private s_raiseFlagData = abi.encodeWithSelector(FlagsDevInterface.raiseFlag.selector, s_arbitrumFlag);
+  bytes constant private s_lowerFlagData = abi.encodeWithSelector(FlagsDevInterface.lowerFlag.selector, s_arbitrumFlag);
+  uint32 constant private s_L2GasLimit = 30000000;
```

Comment:
IMHO this should be configurable. @RodrigoAD any reason for 30 mil value here, seems excessive?